### PR TITLE
Fixed missing `Stringable` interface in the list retrieved by `ReflectionClass::getImmediateInterfaces()`

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/Reflection/ReflectionClass.php [hunk 1]
```diff
@@ -1204,7 +1204,8 @@ public function getImmediateInterfaces(): array
 
         $nodes = $this->node instanceof InterfaceNode ? $this->node->extends : $this->node->implements;
 
-        return array_combine(
+        /** @var array<class-string, self> $interfaces */
```
Comment (Ocramius) [inline src/Reflection/ReflectionClass.php:1207]: Hmm, is this docblock needed at all? :thinking:

Comment (kukulich) [reply src/Reflection/ReflectionClass.php:1207]:
```
ERROR: ArgumentTypeCoercion - src/Reflection/ReflectionClass.php:1218:46 - Argument 1 of Roave\BetterReflection\Reflection\ReflectionClass::addStringableInterface expects array<class-string, Roave\BetterReflection\Reflection\ReflectionClass>, parent type array<string, Roave\BetterReflection\Reflection\ReflectionClass> provided (see https://psalm.dev/193)
        return $this->addStringableInterface($interfaces);
```

Comment (Ocramius) [reply src/Reflection/ReflectionClass.php:1207]: Ah, so `$interfaceName->toString()` is not a `class-string` :-\ 

Never mind then

**Review by Ocramius — APPROVED** [review]
LGTM :+1:

**Pull request merged**